spec/actors/hyrax/actors/generic_work_actor_spec.rb

```diff
@@ -204,11 +201,8 @@
   describe '#update' do
     let(:curation_concern) { create(:generic_work, user: user, admin_set_id: admin_set.id) }
 
-    subject { Hyrax::CurationConcern.actor }
-
     context 'failure' do
       let(:attributes) { {} }
-
```

Comment:
Later versions of rubocop-rspec suggests having blank lines after the subject and final let block.